### PR TITLE
Puzzle - Entry - Is Palindrome

### PR DESCRIPTION
**Pull request by wahidyankf**
This MR will add `isPalindrome` puzzle to this repo.

### CODE REVIEW

File: puzzles/is-palindrome/isPalindrome.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+// -----
+// Run time env (tested on):
+// Node v11.12.0
+// -----
+// Algorithm complexity
```
Comment (wahidyankf) [inline puzzles/is-palindrome/isPalindrome.js:5]: Every solution that is submitted needs to contain its algorithm complexity value. So that we will be able to identify its performance.

File: puzzles/is-palindrome/isPalindrome.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+// -----
+// Run time env (tested on):
+// Node v11.12.0
```
Comment (wahidyankf) [inline puzzles/is-palindrome/isPalindrome.js:3]: We should also identify its node runtime versions, so that the user of this solution will be able to use the appropriate node version to run its solution

File: puzzles/is-palindrome/isPalindrome.test.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
+const isPalindrome = require('./isPalindrome');
```
Comment (wahidyankf) [inline puzzles/is-palindrome/isPalindrome.test.js:1]: A test should also be provided for each puzzle. (to make sure the solution is correct, and all edge cases are covered).
